web/themes/custom/pineapple/sass/components/paragraphs/_image-paragraph.scss

```diff
@@ -1,14 +1,27 @@
 .paragraph--type--image-paragraph {
-  .image-paragraph-image {
-    img {
-      object-fit: cover;
-      width: 100%;
+  .inline-field-label {
+
+    .field__label {
+      font-weight: bold;
+      margin-left: -$unit;
+      font-size: $small-font;
+    }
+
+    .field__value {
+      font-size: $small-font;
     }
   }
```

Comment:
What makes the styling of these labels different to the ones in the rest of the site?

Reply:
You're right. The font size could have been added in the main sass file , but the margin left is necessary here.